mode-apis-base/src/entityModels.ts

```diff
@@ -442,7 +469,15 @@ export interface FetchEntityFilters {
     // An comma separated list of entity classes
     readonly entityClasses?: string | undefined;
     readonly parentId?: string | null | undefined;
+    // To tell the backend whether or not to return only the DIRECT children or includes all descendants.
+    // By default, the back end will only include the entity instances that are direct children of the specified "parentId". If this flag
+    // is set to true, the back end will also includes children, grandchildren, great grandchildren, etc... of the "parentId"
+    readonly recursive?: boolean | undefined;
     readonly attributesQuery?: string | undefined;
+
+    // To have the back end sort the results by the specified "sortField" (attribute name) in the specified "sortOrder"
+    readonly sortField?: string;
+    readonly sortOrder?: SortOrder;
```

Comment:
@bananaumai and @jamesbalcombe83, sorry I renamed the `sortBy` param to `sortField` and all the discussions are gone.

Reply:
Ah actually I think my comment was confusing. the `sortBy` param is actually a composite of `sortField:sortOrder`, not that it should be renamed.

Reply:
I like `sortField` too, it is more accurate.